keyboards/atlantis/ps17/readme.md

```diff
@@ -0,0 +1,31 @@
+# atlantis/ps17
+
+TODO: ADD PHOTO
+![atlantis/ps17](imgur.com image replace me!)
```

Comment:
photo?

Reply:
This is why this PR is in draft (; I'll try to make and upload some soon.

Reply:
Added along with photos of pcb's front/back